### ISSUE TITLE
"FLAC: Could not process single!

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bigman73/practicesharp#25]
```

1. Change .NET Framework of the TestApp to .NET 4.0 or 4.5
2. Execute
3. Error on 'FLAC__stream_decoder_process_single'

This problem do not occurs with .NET 3.5 or lower.
You can reproduce this issue on any app with .NET 4.0 and this DLL included.
```

Original issue reported on code.google.com by `caplat.a...@gmail.com` on 25 May 2015 at 12:57

**Comment by bigman73** [comment]
Fixed in version 2.0.7

**Comment by bigman73** [comment]
NuGet NAudioFlac library is used (.NET 4.5)
https://www.nuget.org/packages/NAudio.Flac/
